scripts/test-check-false-positives.sh

```diff
@@ -23,8 +23,9 @@ function check_expected_errors() {
   local package_root=$1
   local package_name=""
   package_name=$(basename "$1")
+  package_name_manifest=$(cat "$package_root/manifest.yml" | yq -r '.name')
   local expected_errors_file="${package_root%/}.expected_errors"
-  local result_tests="build/test-results/${package_name}-*.xml"
+  local result_tests="build/test-results/${package_name_manifest}-*.xml"
```

Comment:
Test results file names are using the package name defined in the manifest. Example:
`build/test-results/rally_benchmarks-static-1723556393923280393.xml`